core/invoker/src/main/scala/org/apache/openwhisk/core/invoker/NamespaceBlacklist.scala

```diff
@@ -38,6 +38,9 @@ class NamespaceBlacklist(authStore: AuthStore) {
 
   private var blacklist: Set[String] = Set.empty
 
+  // Enforced by couchdb, max limit value for db view query
+  private var couchdb_limit: Int = 268435456
```

Comment:
This should be a `val` and as it's a constant should be written in all caps, like `COUCHDB_LIMIT`.

However, I feel like this should be factored into the couchdb code itself and the `limit` value should have an upper-bound applied there. Technically you do not know in this piece of code that you're handling CouchDB.

Reply:
Fair point!